src/views/enroll-choices/FactorRow.js

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+  'okta',
+  'util/RouterUtil'
+], function (Okta, RouterUtil) {
+
+  return Okta.View.extend({
+    className: 'enroll-factor-row clearfix',
+
+    template: '\
+      <div class="enroll-factor-icon-container">\
+        <div class="factor-icon enroll-factor-icon {{iconClassName}}">\
+        </div>\
+      </div>\
+      <div class="enroll-factor-description">\
+        <h3 class="enroll-factor-label">{{factorLabel}}</h3>\
+        {{#if factorDescription}}\
+          <p>{{factorDescription}}</p>\
+        {{/if}}\
+        <div class="enroll-factor-button"></div>\
+      </div>\
+    ',
+
+    attributes: function () {
+      return { 'data-se': this.model.get('factorName') };
+    },
+
+    children: function () {
+      if (this.model.get('enrolled')) {
+        return [['<span class="icon success-16-green"></span>', '.enroll-factor-label']];
+      }
+      /*else if(this.model.get('enrollment') === 'REQUIRED'){
+       return [['<span class="icon success-16-gray"></span>', '.enroll-factor-label']];
+       }*/
```

Comment:
Not sure if it is a good architectural solution to have the "required "logic in the `FactorRow.js`, not in the `RequiredFactorRow.js`, since we have two different `FactorList.js` and `RequiredFactorList.js`.